### PR TITLE
Pass verbose argument to LLMChains when using *DocumentsChain

### PR DESCRIPTION
**Pull request by parthchadha**
When using chains such as Summarization chain (`load_summarize_chain`), the verbose flag wasn't propagated to the `LLMChain`.

### CODE REVIEW

File: langchain/chains/qa_with_sources/__init__.py [hunk 1]
```diff
@@ -49,9 +49,13 @@ def _load_map_reduce_chain(
...
-    map_chain = LLMChain(llm=llm, prompt=question_prompt)
+    map_chain = LLMChain(
+        llm=llm, prompt=question_prompt, verbose=kwargs.get("verbose", False)
+    )
     _reduce_llm = reduce_llm or llm
-    reduce_chain = LLMChain(llm=_reduce_llm, prompt=combine_prompt)
+    reduce_chain = LLMChain(
+        llm=_reduce_llm, prompt=combine_prompt, verbose=kwargs.get("verbose", False)
+    )
     combine_document_chain = StuffDocumentsChain(
```
Comment (hwchase17) [inline langchain/chains/qa_with_sources/__init__.py:59]: probably want to pass it in here as well

File: langchain/chains/qa_with_sources/__init__.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def _load_stuff_chain(
     document_variable_name: str = "summaries",
     **kwargs: Any,
 ) -> StuffDocumentsChain:
-    llm_chain = LLMChain(llm=llm, prompt=prompt)
+    llm_chain = LLMChain(llm=llm, prompt=prompt, verbose=kwargs.get("verbose", False))
```
Comment (hwchase17) [inline langchain/chains/qa_with_sources/__init__.py:31]: given the amount of times we do `kwargs.get()`, lets maybe just make it an explicit kwarg in thsi function, eg `verbose=False` on line 29

**Review by hwchase17 — COMMENTED** [review]
thanks for this!

only commented on qa_with_sources, but same comments go for other two use cases as well

File: langchain/chains/qa_with_sources/__init__.py (hunk 1 above)
Comment (parthchadha) [reply langchain/chains/qa_with_sources/__init__.py:57]: `StuffDocumentsChain` takes in `reduce_chain` llm which has verbose arguments set.

File: langchain/chains/qa_with_sources/__init__.py (hunk 2 above)
Comment (parthchadha) [reply langchain/chains/qa_with_sources/__init__.py]: Thanks for the suggestion, made verbose an explicit kwarg.

**Review by hwchase17 — APPROVED** [review]

**Pull request merged**